src/main/java/meditracker/command/UntakeCommand.java

```diff
@@ -84,6 +74,7 @@ public void execute() {
                     "Unable to increase Medication quantity as object not found");
```

Comment:
I think can just say medication is not found. "increase" and "object" does not mean much to the user.

Reply:
Partially resolved in e3efe25
I believe user should understand the context of why this warning is shown to them so I have reworded slightly to be more user friendly

Reply:
I mean in the context of untaking a medication (which usually happens when they accidentally check off the wrong meds), the message saying "increase medication quantity" is not a deliberate/intentional from user. 
So it could lead to the user questioning "why am I trying to increase medication quantity? I am just trying to uncheck the medicine."
So in a sense, that information is not necessary to inform them.

Reply:
Resolved in 1d7d633